test/e2e/e2e_suite_test.go

```diff
@@ -433,8 +445,8 @@ func resolveKubernetesVersion(config *clusterctl.E2EConfig, versions semver.Vers
 	config.Variables[varName] = v
 }
 
-// getAllCAPIImageSkus returns all skus for the capi offer under the "cncf-upstream" publisher.
-func getAllCAPIImageSkus(ctx context.Context, location string) []string {
+// getImageSkusInOffer returns all skus for an offer that have at least one image.
+func getImageSkusInOffer(ctx context.Context, publisher, offer, location string) []string {
```

Comment:
the order of these args is different the `imagesClient.ListSkus(ctx, location, publisher, offer)` which was a bit confusing when reading. This is minor but could help clarity.

Reply:
true, let me fix that to make them match

Reply:
fixed